usr/share/rear/layout/recreate/default/200_run_layout_code.sh

```diff
@@ -29,13 +53,30 @@ unset choices
 choices[0]="Rerun disk recreation script ($LAYOUT_CODE)"
 choices[1]="View '$rear_workflow' log file ($RUNTIME_LOGFILE)"
 choices[2]="Edit disk recreation script ($LAYOUT_CODE)"
-choices[3]="View original disk space usage ($original_disk_space_usage_file)"
-choices[4]="Use Relax-and-Recover shell and return back to here"
-choices[5]="Abort '$rear_workflow'"
-prompt="The disk layout recreation script failed"
+choices[3]="Again wipe those disks: $DISKS_TO_BE_WIPED"
```

Comment:
What if `$DISKS_TO_BE_WIPED` is an empty string (the default)? Won't it print just
```
Again wipe those disks: 
```
which would be very confusing?

Reply:
I know it and this is how it is since the beginning
also at other code places as far as I remember.
This needs to be cleaned up.
But not with this pull request.

Reply:
Since after ReaR 2.7 release DISKS_TO_BE_WIPED
is the empty string th default.conf, cf.
https://github.com/rear/rear/pull/2859
but this means DISKS_TO_BE_WIPED will usually
be set to an non-empty string via the 'else' clause in
layout/recreate/default/120_confirm_wipedisk_disks.sh

Only if the create_disk_label function is not called
for a disk in diskrestore.sh then DISKS_TO_BE_WIPED
will stay empty and this case is normally an error case
but I did not implement handling of this case via
empty DISKS_TO_BE_WIPED because there never was
handling for the case that the create_disk_label function
is not called for a disk in diskrestore.sh
and empty DISKS_TO_BE_WIPED is onyl a symptom
but not the cause of it.

Reply:
> Currently it does not work to run diskrestore.sh
> then "Again wipe the disks in DISKS_TO_BE_WIPED"
> and then re-run diskrestore.sh
> because the second run of diskrestore.sh does nothing
> because all is marked as already done by the first run.

I suppose a good fix would be to unmark everything as done when this choice is selected, because wiping disks undoes it. The only complication is if DISKS_TO_BE_WIPED covers only a part of what has been done so far.

Reply:
That's my plan to "just unmark" all what is marked as done.

But I don't know yet if it is easy to implement in practice
(I have no practical experience with the relevant code parts).

Perhaps something like `rm -rf $LAYOUT_TOUCHDIR` as in
layout/prepare/default/010_prepare_files.sh
is all what needs to be done as far as I understand
from plain looking at the code what `create_component()`
and `component_created()` in lib/layout-functions.sh
seem to do.